test/plug/static_test.exs

```diff
@@ -238,6 +238,88 @@ defmodule Plug.StaticTest do
     end
   end
 
+  describe "range request" do
+    test "serves entire file if range is 0-" do
+      conn = conn(:get, "/public/fixtures/static.txt", [])
+             |> put_req_header("range", "bytes=0-")
+             |> call
+
+      assert conn.status == 200
+      assert conn.resp_body == "HELLO"
+      assert get_resp_header(conn, "content-type")  == ["text/plain"]
+    end
+
+    test "serves requested range of file starting from byte 0" do
+      conn = conn(:get, "/public/fixtures/static.txt", [])
+             |> put_req_header("range", "bytes=0-1")
+             |> call
+
+      assert conn.status == 206
+      assert conn.resp_body == "HE"
+      assert get_resp_header(conn, "content-type")  == ["text/plain"]
+    end
+
+    test "serves tail of file if range is -n" do
+      conn = conn(:get, "/public/fixtures/static.txt", [])
+             |> put_req_header("range", "bytes=-3")
+             |> call
+
+      assert conn.status == 206
+      assert conn.resp_body == "LLO"
+      assert get_resp_header(conn, "content-type")  == ["text/plain"]
+    end
+
+    test "returns 416 if range does not contain either start or end" do
+      exception = assert_raise Plug.Static.InvalidRangeError,
+                               "invalid range for static asset",
+        fn ->
+          conn(:get, "/public/fixtures/static.txt", [])
+            |> put_req_header("range", "bytes=-")
+            |> call
+        end
+
+      assert Plug.Exception.status(exception) == 416
+    end
+
+    test "returns 416 if range is contains non-integers" do
+      exception = assert_raise Plug.Static.InvalidRangeError,
+                               "invalid range for static asset",
+        fn ->
```

Comment:
Please start `fn ->` in the previous line and outdent the whole function 2 spaces. The same for the examples below.